src/components/appmodallogin.main.less

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+
+.login-modal-content {
+  #login_modal_form {
+    span {
+      display: inline-block;
+      margin-bottom: 5px;
+      font-weight: bold;
+    }
+  }
+
+  #simplemodal-container {
+    background-color: #fff;
+    border: 8px solid #444;
+    padding: 12px;
+    width: auto !important;
+    height: auto !important;
+  }
+
+  .btn-auth-login {
+    display: inline-block;
+    padding: 6px 12px;
+    margin-bottom: 0;
+    font-size: 14px;
+    font-weight: normal;
+    line-height: 1.428571429;
+    text-align: center;
+    vertical-align: middle;
+    cursor: pointer;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    white-space: nowrap;
+    user-select: none;
+    padding: 10px 16px;
+    font-size: 18px;
+    line-height: 1.33;
+    border-radius: 6px;
+    border-radius: 2px;
+    box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.2);
+    margin-bottom: 2px;
+    color: #ffffff;
+    background-color: @mainColor;
+    border-color: @mainColor;
```

Comment:
Border color is already defined at line 51. You can replace `transparent` with `@mainColor` on that line and remove this line.

Reply:
replaced.